UC-SOAR/PeripheralDriversSubmodule
# Noah/Barometer_Drivers

Created the drivers for MS5611 and MS5607 Barometers

To test:
* Create a loop that periodically gets samples from the barometers and logs/records them
* Verify the readings are correct

To integrate:
* include the required header file
* declare an object of the class with the spi handle and required pins in the constructor
* call getSample() to return a data structure containing temp and pressure readings
